llvm/unittests/Support/TrailingObjectsTest.cpp

```diff
@@ -14,19 +14,19 @@
 using namespace llvm;
 
 namespace {
-// This class, beyond being used by the test case, a nice
-// demonstration of the intended usage of TrailingObjects, with a
-// single trailing array.
-class Class1 final : protected TrailingObjects<Class1, short> {
+// This class, beyond being used by the test case, a nice demonstration of the
+// intended usage of TrailingObjects, with a single trailing array.
+class Class1 final : private TrailingObjects<Class1, short> {
```

Comment:
not quite following why this became private?

Reply:
The comment in TrailingObjects.h says:

```
/// All the methods implemented by this class are intended for use
/// by the implementation of the class, not as part of its interface
/// (thus, private inheritance is suggested).
```

Hence making this private as the test case should also demonstrate the recommended usage of this class.

Reply:
But I'll do that as a follow-on change and not here. so reverted.